web/vocab/TERMS.md

```diff
@@ -0,0 +1,109 @@
+     - Sense
+     - Practice
+     - Object
+     - Person
+     - Researcher
+     - Place
+
+## Event
+- A place and time at which some action was observed or is known to have happened.  See http://schema.org/Event
+
+## Place
+- A location, made knowable by some label (metaphysical, imaginary or not on planet Earth, ex. "Legion of Doom") and/or earthbound coordinates.  See http://schema.org/Place.
+
+## Person
+- An imaginary or real human being.  See http://schema.org/Person
+
+## Researcher
+- All the qualities of a Person except information is limited to name only and the context is confined to partners in the Lived Religion project.
+
+## Sense
+- One of the five common human senses: Sight, Sound, Smell, Touch, Taste.  See http://id.worldcat.org/fast/1112562
+
+## Practice
+- An action being performed by individual(s) based on religious guidance.  See http://id.worldcat.org/fast/1149637
+
+## Object
+- Observable physical matter. See http://www.ebu.ch/metadata/ontologies/ebucore/ebucore#PhysicalResource
+
+## RelatedSenses
+- Aggregates the various Senses under an ExperienceEvent.
+
+## RelatedPractices
+- Aggregates the various Practices under an ExperienceEvent.
+
+## RelatedObjects
+- Aggregates the various Objects under an ExperienceEvent.  See http://www.ebu.ch/metadata/ontologies/ebucore/ebucore#hasArtefactRelatedPhysicalResource
+
+## Sight
+- A Person's Sense of Sight, noting the person and related phenomenon causing the sense.  See http://id.worldcat.org/fast/1167852  
+
+## Sound
+- A Person's Sense of Sound, noting the person and related phenomenon causing the sense. See http://id.worldcat.org/fast/1127099
+
+## Feel
+- A Person's Sense of Feel, noting the person and related phenomenon causing the sense. See http://id.worldcat.org/fast/1153089
+
+## Smell
+- A Person's Sense of Smell, noting the person and related phenomenon causing the sense.  See http://id.worldcat.org/fast/1121562
+
+## Taste
+- A Person's Sense of Taste, noting the person and related phenomenon causing the sense. See http://id.worldcat.org/fast/1143475
+
+## Singing
+- The act of singing being performed by a Person or Persons.  See http://id.worldcat.org/fast/1119311
+
+## Sitting
+- The act of sitting being performed by a Person or Persons. See http://id.worldcat.org/fast/1073252
+
+## Kneeling
+- The act of kneeling being performed by a Person or Persons. See http://id.worldcat.org/fast/1073252
+
+## Standing
+- The act of standing being performed by a Person or Persons. See http://id.worldcat.org/fast/1073252
+
+## Traveling
+- The act of traveling being performed by a Person or Persons.  See http://id.worldcat.org/fast/1155558
+
+## Dancing
+- The act of dancing being performed by a Person or Persons.  See http://id.worldcat.org/fast/887402
+
+## Working
+- The act of work to produce some predefined outcome being performed by a Person or Persons.
+
+## Cooking
+- The act of preparing Food. See http://id.worldcat.org/fast/1754966 
+
+## ClotheAction
```

Comment:
I think "-Action" is redundant here.

Reply:
+1

Reply:
Changing this means the same changes in context as well, of course.